fastparquet/test/test_dataframe.py

```diff
@@ -33,6 +34,7 @@ def test_empty():
     assert len(views) == 5
 
 
+@pytest.mark.xfail(reason="df._data is going away")
```

Comment:
They replaced `_data` with `_mgr`. There is also a conversation in this PR  about `hash_pandas_object `: https://github.com/dask/dask/pull/10081.

Reply:
How far back does _mgr exist?

Reply:
Looks like the renaming happened a long time ago. The PR belongs to 1.1 milestone:
https://github.com/pandas-dev/pandas/pull/27254
But the deprecation is very recent.

Reply:
https://github.com/dask/fastparquet/pull/857